ui/src/components/settings/sub_pages/profile/mod.rs

```diff
@@ -146,6 +149,10 @@ pub fn ProfileSettings(cx: Scope) -> Element {
         special_chars: None,
     };
 
+    let did_string = state.read().did_key().to_string();
+    let mut did_short = "#".to_string();
+    did_short.push_str(&did_string[did_string.chars().count() - 5..]);
```

Comment:
if the state is `Identity`, you can extract the short id from it instead by doing `state.read().short_id()`. Do note this would yield the last 8 characters of the did key if you decided to do that.

Reply:
Let's go this path just to keep things standardized.